jwst_mast_query/jwst_query.cfg

```diff
@@ -0,0 +1,78 @@
+# specify the instrument. The default set in the code is nircam
+instrument: nircam
+
+# define the output filename
+outrootdir: $JWSTDOWNLOAD_OUTDIR
+outsubdir: 
+skip_propID2outsubdir: False
+skip_check_if_outfile_exists: False
```

Comment:
Documentation might help here too. Or somewhere, talking about how the check works. Does it check for all files from a given query? Or just a single example file?